contents/handbook/people/team-structure/core-experience.md

```diff
@@ -3,4 +3,35 @@ title: Team Core Experience
 sidebar: Handbook
 showTitle: true
 hideAnchor: true
----
\ No newline at end of file
+---
+
+## People
+
+- [Eric Duong (Team Lead, Full Stack Engineer)](/handbook/company/team/#eric-duong-software-engineer)
+- [Paolo D'Amico (Product Manager)](/handbook/company/team/)
+- [Buddy Williams, Full Stack Engineer](/handbook/company/team/#buddy-williams-software-engineer)
+- Sam Winslow, Full Stack Engineer
+- [Li Yi Yu, Full Stack Engineer]((/handbook/company/team/#li-yi-yu-software-engineer))
+
+## Mission
+
+The most easy to use and useful analytics tool/experience to discover insights about your product and users
```

Comment:
Easy to use and useful sounds a bit strange, thinking of alternatives, how about something like "The easiest way to discover insights about products and users"?

Reply:
I agree with Paolo here, also maybe adding "To create" to it, like "To create the easiest way to discover insights about products and its users" since that flows better with "Our mission is to create the easiest way" vs "Our mission is the easiest way"

Reply:
yup done